### DISCUSSION TITLE
BirdNET-Pi and tar1090 can run paralel in Raspberry Pi 4?

### DISCUSSION THREAD

**Discussion by kinteki** [mcguirepr89/BirdNET-Pi#404] [Q&A]
Hi,

I've installed first _tar1090_ ( worked properly) and then  BirdNET-Pi. So now at the _tar1090_ local IP's appeared the birdnet.

![image](https://user-images.githubusercontent.com/107460149/175519747-2c5be96b-478d-4b64-a94a-fc731975c0c6.png)

How can I reach that this work paralell together?

Bests, 
Zoltan

**Comment by mcguirepr89** [comment]
https://github.com/mcguirepr89/BirdNET-Pi/discussions/377#discussioncomment-3016979
